debian/gstd.postinst

```diff
@@ -13,11 +16,11 @@ if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-decon
 		if [ ! -e "$dir" ]; then
 			if mkdir "$dir" 2>/dev/null; then
 				chown "$user":"$group" "$dir"
-				chmod -R "$mode" "$dir"
+				chmod "$mode" "$dir"
 			fi
 		else
                         chown "$user":"$group" "$dir"
-                        chmod -R "$mode" "$dir"		
+                        chmod "$mode" "$dir"
```

Comment:
Is recursive not longer needed?

Reply:
`chmod -R` is prohibited in Debian packages. it generates the error code `maintainer-script-should-not-use-recursive-chown-or-chmod`

It also wasn't required because `/var/run` and `var/log` are empty when gstd is installed